LeftPanel.qml

```diff
@@ -674,10 +663,20 @@ Rectangle {
             id: daemonProgressBar
             anchors.left: parent.left
             anchors.right: parent.right
-            anchors.bottom: parent.bottom
+            anchors.bottom: networkStatus.top
             syncType: qsTr("Daemon") + translationManager.emptyString
             visible: networkStatus.connected
             height: 62
         }
+        MoneroComponents.NetworkStatusItem {
```

Comment:
Nitpick but please add an empty line between } and MoneroComponents.NetworkStatusItem.